AdobeStockImageAdminUi/view/adminhtml/web/template/grid/column/preview/related.html

```diff
@@ -16,12 +16,12 @@
                 }
             }">
         <ul class="tabs-horiz">
-            <li>
+            <li data-bind="visible: (typeof relatedImages().series[$row().id] === 'undefined' || relatedImages().series[$row().id] !== false)">
```

Comment:
What do you think about using `if` attribute binding instead of `data-bind="visible` - shorter and shouldn't render the element at all